File: Lab3/src/main/java/domain/Inscriere.java
package domain;

public class Inscriere extends Entity<Long>{
    private Participant participant;
    private Proba proba;

    public Inscriere(Participant participant, Proba proba) {
        this.participant = participant;
        this.proba = proba;
    }

    public Inscriere(){}

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public Proba getProba() {
        return proba;
    }

    public void setProba(Proba proba) {
        this.proba = proba;
    }
}

File: Lab3C#/domain/Inscriere.cs
namespace Lab3.domain;

public class Inscriere : Entity<long>
{
    public Participant Participant { get; set; }
    public Proba Proba { get; set; }

    public Inscriere(Participant participant, Proba proba)
    {
        Participant = participant;
        Proba = proba;
    }

    public Inscriere() { }
}